src/Hooks.cpp

```
@@ -198,23 +198,25 @@ namespace Hooks
-		} else {
-			Events::FrameEvent e;
-			e.gamePaused = a_this->freezeTime;
-			Events::Sources::FrameEventSource::GetSingleton()->SendEvent(&e);
+			if (quitGame) {
+				Events::ShutdownEvent e;
+				Events::Sources::ShutdownEventEventSource::GetSingleton()->SendEvent(&e);
+				return;
+			}
 		}
+
+		Events::FrameEvent e;
```

Clang (14.0.6)

[warning] uninitialized record type: 'e'

(cppcoreguidelines-pro-type-member-init)